tests/OracleTest.php

```diff
@@ -33,8 +33,8 @@ public function connect($ver = '')
     {
         return new \atk4\dsql\Connection(array_merge([
             'connection'       => new \PDO('sqlite::memory:'),
-            'query_class'      => 'atk4\dsql\Query_Oracle'.$ver,
-            'expression_class' => 'atk4\dsql\Expression_Oracle',
```

Comment:
`\atk4\dsql\Expression_Oracle` class seems to be even not presented

Reply:
Should remove that expression_class line (will use default Expression class then), but query_class line should be here!

Reply:
done, of course - the first one was refactored only, but GH is showing 1 red/removed line and 1 added